.github/workflows/test.yml

```diff
@@ -43,11 +43,14 @@ jobs:
         python -m pip install --upgrade pip
         pip install pytest
         pip install -r test_requirements/${{ matrix.requirements-file }}
+        pip install -r test_requirements/postgres.txt
         python setup.py install
 
     - name: Test with django test runner
       run: |
         python manage.py test
+      env:
+        DATABASE_URL: postgres://postgres:postgres@127.0.0.1/postgres
```

Comment:
This is a great catch! Thanks for spotting this.

Reply:
We have to think about people that want to run `./manage.py test` on their computer, just using SQLite database, or using the postgreSQL database. If you include `mysqlclient==2.0.3` in `requirements_base.txt` file, the test will crash because it needs mySQL ou mariadDB to be installed on the computer:
    OSError: mysql_config not found
    mysql_config --version
    mariadb_config --version
    mysql_config --libs

Reply:
@fp4code Alright, let me think about it.

Reply:
Pleas think about a Django-CMS tester. He will tray to to a test (using just default sqlite).
`./manage.py test` Result if we do not split `mysqlclient` and `psycopg2` out or requirement_base.txt file, the test will crash. Very bad for Django-CMS fame!